gpytorch/kernels/multitask_kernel.py

```diff
@@ -38,21 +38,24 @@ def forward_diag(self, x1, x2):
         in :class:`gpytorch.kernels.Kernel` because we need to take the Kronecker product of the diagonals of the
         base data kernel and the task kernel.
         """
-        x1 = x1.transpose(-2, -3)
-        x2 = x2.transpose(-2, -3)
         task_indices = torch.arange(self.n_tasks, device=x1.device).long()

```

Comment:
`long()` shouldn't be necessary here

Reply:
torch.arange doesn't produce a long tensor by default:
```python
In [2]: torch.arange(5).dtype
Out[2]: torch.float32
```

Reply:
huh. here's my output:
```
In [2]: torch.arange(5).dtype
Out[2]: torch.int64
```
I guess there must have been a change on pytorch master... let's be safe for now